packages/components/src/text/stories/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { Text } from '../Text.styles';
+
+export default {
+	title: 'components|Text',
+	component: Text,
+};
+
+export const TitleLarge = () => <Text variant="title.large" as="h1">Title Large</Text>;
```

Comment:
@jameslnewell  Thank you for making all of these stories! 🙏 

After poking around at them, I had an thought to maybe improve the Text story UX a bit.
Similar to what we have in the other stories, what are you thoughts on rigging up knobs for a `_default` story?

Another cool idea would maybe be a page that shows all of the sizes at once, example:
https://wordpress.github.io/gutenberg/?path=/story/components-icon--sizes

That can provide an experience similar to Type Scale documentation from other systems, example:
https://material.io/design/typography/the-type-system.html#type-scale

Granted, it doesn't need to be as detailed as that 😊

Reply:
I've created `_default` story with all the variants in it. 

I don't know if there's much value for knobs at the moment... maybe in a future PR when I add color and other props?